core/base/test/TStringTest.cxx

```diff
@@ -0,0 +1,18 @@
+#include "gtest/gtest.h"
+
+#include "ROOTUnitTestSupport.h"
+
+#include "TString.h"
+
+TEST(TString, Basics)
+{
+   TString *s = nullptr;
+   EXPECT_STREQ("", s);
```

Comment:
ah, this line doesn't belong here. The tricky part is that `TString *s = nullptr` does _not_ construct a `TString` object, it constructs a _pointer_ to a `TString` object, and that pointer is "null" at this point (i.e. it points to nothing).

Later, `s = new TString` actually creates a `TString` object and makes `s` point to it. Python doesn't have pointers, C++ does!

Reply:
Ohh okay.

Reply:
So instead of 
```
TString *s = nullptr;
```
I write:
```
s = new TString("Test", -5);
```
?